### PR TITLE
Stagger thread creation when affinity not specified

### PR DESCRIPTION
**Pull request by randprint**
feels more smooth ¯\_(ツ)_/¯

### CODE REVIEW

**Comment by TGLTClutch** [comment]
is this a professional PR I'm seeing. 

👀👀👀

File: src/xenia/kernel/xam/user_profile.cc [hunk 1]
```diff
@@ -18,8 +18,8 @@ namespace kernel {
 namespace xam {
 
 UserProfile::UserProfile() {
-  xuid_ = 0xBABEBABEBABEBABE;
-  name_ = "User";
+  xuid_ = 0xE000BABEBABEBABE;
```
Comment (Razzile) [inline src/xenia/kernel/xam/user_profile.cc]: This change isn't related to the PR right?

Comment (randprint) [reply src/xenia/kernel/xam/user_profile.cc]: oof no, clicked the wrong button

**Comment by d4rkne55** [comment]
What happened with this?

**Pull request closed without merging**